LandSandBoat/server
# Perpetuance job ability implementation

**_I affirm:_**

## What does this pull request do?



Logic for perpetuance increasing enhancing duration was implemented but the status effect and removal logic was missing. Added the ability to the db with the correct light arts flags and removal of the status effect on the next white magic enhancing spell.

Also fixed light arts status effect flags as it was not properly being removed when logging out. I double checked that behavior on retail to make sure I wasn't crazy.

## Steps to test these changes



* Cast enhancing magic on yourself (e.g. Regen)
* Use Lights Arts -> Perpetuance
* Cast same enhancing magic on yourself
* Observe the duration is doubled compared to the first cast
